### PR TITLE
add instructions for configuring e2e test runner on a fresh deployment

### LINKED ISSUE #1236: Write e-2-e test runner setup guide
How to set this up on a freshly deployed system

/assign

### PR DESCRIPTION
**Pull request by mikehelmick**
Fixes #1236 

## Proposed Changes

* docs - how to configure e2e test runner

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: docs/production.md [hunk 1]
```diff
@@ -412,6 +413,68 @@ The verification server uses the Google Identity Platform for authorization.
...
+
+Select `Signing Keys` from the drop down menu
+
+![select signing keys](images/e2e/image04.png)
+
+Create a new (initial) realm specific signing key
+
+![create realm signing keys](images/e2e/image05.png)
+
+Set the issuer and audience values (`iss`/`aud`) according to your domain settings, and click `Save realm certificate settings`. You will need this information
```
Comment (sethvargo) [inline docs/production.md:438]: What does "according to your domain settings" mean in this context?

**Review by sethvargo — APPROVED** [review]
/hold 
for one q

File: docs/production.md (hunk 1 above)
Comment (mikehelmick) [reply docs/production.md]: added suggested values

**Review by mikehelmick — COMMENTED** [review]
/hold cancel

**Pull request merged**